src/app/shared/case-editor/case-edit.component.spec.ts

```diff
@@ -188,11 +188,7 @@ describe('CaseEditComponent', () => {
   beforeEach(() => {
   });
 
-  it('should render an event trigger header', () => {
-    let header = de.query(By.directive(EventTriggerHeaderComponent));
-    expect(header).toBeTruthy();
-    expect(header.componentInstance.eventTrigger).toEqual(EVENT_TRIGGER);
-  });
+  // Moved this test case to case-edit-page.component
```

Comment:
Should we also remove unused mock component?
`   let EventTriggerHeaderComponent: any = MockComponent({
     selector: 'ccd-event-trigger-header',
     inputs: ['eventTrigger']
   });`